dashboard/db/migrate/20170127183604_create_sign_ins.rb

```diff
@@ -0,0 +1,13 @@
+class CreateSignIns < ActiveRecord::Migration[5.0]
+  def change
+    create_table :sign_ins do |t|
+      # The user to which the sign-in applies.
+      t.belongs_to :user, null: false
+      # The timestamp of the sign-in. The column is indexed so as to be able to
+      # quickly compute 7-DA and 30-DA counts.
+      t.timestamp :sign_in_at, null: false, index: true
```

Comment:
Naming this column `created_at` or `created_on` would allow you to leverage ActiveRecord's existing [Timestamp](https://github.com/rails/rails/blob/21f0136fc7c389751fec975823f99695eb7db3f9/activerecord/lib/active_record/timestamp.rb) functionality without additional callback configuration.